CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Project root is simplified/refactored [#1738](https://github.com/KomodoPlatform/atomicDEX-API/pull/1738)
 - Created base image to provide more glibc compatible pre-built binaries for linux [#1741](https://github.com/KomodoPlatform/atomicDEX-API/pull/1741)
 - Set default log level as "info" [#1747](https://github.com/KomodoPlatform/atomicDEX-API/pull/1747)
+- Detect and resolve chain-reorg [#1728](https://github.com/KomodoPlatform/atomicDEX-API/pull/1728)
```

Comment:
Please write a more informative change log, it's not clear that this is done only for spv and block headers sync.